libc-test/build.rs

```diff
@@ -2444,6 +2444,10 @@ fn test_emscripten(target: &str) {
 
     let mut cfg = ctest_cfg();
     cfg.define("_GNU_SOURCE", None); // FIXME: ??
+    cfg.flag("-pthread");
+    cfg.flag("-mbulk-memory");
+    cfg.flag("-matomics");
+    cfg.flag("-Wl,--shared-memory");
```

Comment:
I don't think these flags are needed. The function should be available in the header files even without these flags.

Reply:
Emscripten says those are needed and I can only compile with `-pthread`. Don't know about the other flags though.

Reply:
Removed all :)